src/parser/core/modules/PetTimeline.tsx

```diff
@@ -92,6 +92,7 @@ export default class PetTimeline extends Module {
 
 			// TOOD: Type cooldowns so this cast isn't required
 			const parentRow: ContainerRow = this.cooldowns.getActionTimelineRow(summonAction)
+			parentRow.order = -99
```

Comment:
Looks ok but what's this change for?

Reply:
Since the pet group gets ordered at the top in the other two cases, I think it makes sense to do the same thing for this case.

Reply:
Since the pet group gets ordered at the top in the other two cases, I think it makes sense to do the same thing for this case.